cli/src/pcluster/config/update_policy.py

```diff
@@ -455,3 +504,12 @@ def condition_checker_shared_storage_update_policy(change, patch):
     )
     + ". If you need this change, please consider creating a new cluster instead of updating the existing one.",
 )
+
+# Block update if cluster has a managed Fsx for Lustre FileSystem, otherwise fallback to QueueUpdateStrategy
+UpdatePolicy.MANAGED_FSX = UpdatePolicy(
+    name="MANAGED_FSX",
+    level=5,
```

Comment:
I would say level is > 5 because it's more restrictive than QUEUE_UPDATE_STRATEGY